README.md

```diff
@@ -176,6 +176,7 @@ TradersClub | https://tc.com.br
 TurboSys | http://turbosys.com.br/
 Unbanked | https://www.unbanked.com.br/
 Valtech | https://www.valtech.com/pt-br/carreiras/vagas/?countryBrazil
+ValueCoders | https://www.valuecoders.com/
```

Comment:
Hi! I didn't find anything about remote jobs on this website: https://www.valuecoders.com/careers

Do you have a different link where I can find that information?